src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -32,18 +40,57 @@ const MainNavDroplist = Droplist.extend({
     },
   },
   picklistOptions() {
-    const currentOrg = Radio.request('bootstrap', 'currentOrg');
     return {
       className: 'picklist app-nav__picklist',
-      headingText: currentOrg.get('name'),
+      itemClassName: 'flex flex-align-center app-nav__picklist-item',
+      lists: [{
+        collection: this.collection,
+        itemTemplate: hbs`
+          {{fa icon.type icon.icon classes=icon.classes~}}
+          <span>{{formatMessage text}}</span>
+        `,
+      }],
+    };
+  },
+  picklistEvents: {
+    'picklist:item:select': 'onSelect',
+  },
+  onSelect({ model }) {
+    model.get('onSelect')();
+  },
+});
+
+const AdminToolsDroplist = Droplist.extend({
+  popWidth: '248px',
+  position() {
+    return {
+      top: window.innerHeight - 12,
+      left: 164,
+    };
+  },
+  viewOptions: {
+    tagName: 'div',
+    className: 'flex flex-align-center app-nav__bottom-button',
+    template: hbs`{{far "ellipsis"}}{{ @intl.globals.appNav.appNavViews.adminToolsDroplist.adminTools }}`,
+    templateContext() {
```

Comment:
I don't think this context is used?

Reply:
Correct, it's not. Forgot to remove it. Removed it in last push.